src/usr/local/www/includes/functions.inc.php

```diff
@@ -127,11 +122,11 @@ function get_hwtype() {
 	return;
 }
 
-function get_mbuf(&$mbuf, &$mbufpercent) {
-	$mbufs_output=trim(`/usr/bin/netstat -mb | /usr/bin/grep "mbuf clusters in use" | /usr/bin/awk '{ print $1 }'`);
-	list($mbufs_current, $mbufs_cache, $mbufs_total, $mbufs_max) = explode("/", $mbufs_output);
-	$mbuf = "{$mbufs_total}/{$mbufs_max}";
-	$mbufpercent = ($mbufs_max > 0) ? round(($mbufs_total / $mbufs_max) * 100, 0) : "NA";
+function get_mbuf() {
+	$mbufs_output=trim(`/usr/bin/vmstat -z | /usr/bin/grep mbuf_cluster | /usr/bin/grep -v debugnet | /usr/bin/awk '{print $3,$4,$5}'`);
```

Comment:
This does not work properly in several cases. The output is not guaranteed to have spaces between columns.

For example:
```
: /usr/bin/vmstat -z | /usr/bin/grep mbuf_cluster | /usr/bin/grep -v debugnet 
mbuf_cluster:          2048,1000000,   12028,    1434,  144434,   0,   0,   0

: /usr/bin/vmstat -z | /usr/bin/grep mbuf_cluster | /usr/bin/grep -v debugnet | /usr/bin/awk '{print $3,$4,$5}'
12027, 1435, 144434,
```

If you want to try going that route, call `vmstat -z --libxo=json` and process that output to find the exact values you want accurately. Or to narrow that a bit:

```
: /usr/bin/vmstat -z --libxo=json | /usr/local/bin/jq '."memory-zone-statistics".zone.[] | select(.name=="mbuf_cluster")'
{
  "name": "mbuf_cluster",
  "size": 2048,
  "limit": 1000000,
  "used": 12028,
  "free": 1434,
  "requests": 144620,
  "fail": 0,
  "sleep": 0,
  "xdomain": 0
}
```

Reply:
My bad, you're right - I noticed this one when testing but the fix slipped away somewhere in between the testing and commiting.
One idea was to use comma delimiter with awk, but then whitespaces become potentially problematic (output needs to be trimmed). I'll take a look at json output and try to find the cleanest/fastest solution.